src/geo/transform.js

```diff
@@ -1342,7 +1327,7 @@ class Transform {
             mercatorYfromLat(lat) * worldSize);
 
         // transform the globe from reference coordinate space to world space
-        const posMatrix = mat4.identity(new Float64Array(16));
+        const posMatrix = mat4.identity(new Float32Array(16));
```

Comment:
Do you think single precision is enough here? In most cases intermediate matrix calculations should be done with double precision numbers.

Reply:
No, I think it should be done later (right before passing it as a uniform).